src/project/mesa3d_desktop_intel.rs

```diff
@@ -164,7 +195,15 @@ impl Project for Mesa3DDesktopIntel {
         if target.ends_with("libanv_common.a") {
             libs.push("hwvulkan_headers");
         }
+        if target.ends_with("libEGL_mesa.so.1.0.0") {
+            libs.push("libnativebase_headers");
+        }
         module
+            .add_prop("soc_specific", soc_specific)
+            .add_prop(
+                "relative_install_path",
+                SoongProp::Str(relative_install_path.to_string()),
```

Comment:
I've made the choice (randomly?) to always use `String::from` and never `.to_string()` in `ninja-to-soong`.

Reply:
Gotcha, fixed.